test/utils/browser-music.js

```diff
@@ -1,12 +1,14 @@
 // Load in dependencies
-var Asserter = require('wd/lib/asserters').Asserter;
+var asserters = require('wd/lib/asserters');
+var Asserter = asserters.Asserter;
 
 // Define our helpers
 exports.playAnything = function () {
   before(function playAnythingFn (done) {
     // Find and click the I'm Feeling Lucky mix
     var browser = this.browser;
-    browser.waitForElementByCssSelector('[data-id=shuffle-my-library]', 2000, 100, function handleElement (err, el) {
+    browser.waitForElementByCssSelector('[data-id=shuffle-my-library]', asserters.isDisplayed,
+    10000, 100, function handleElement (err, el) {
```

Comment:
Yeah, wasn't quite sure what to do their, your linting passed on this so I assumed it was correct, I'll make that change :D